.github/workflows/pkgdown.yaml

```diff
@@ -13,56 +13,142 @@ name: pkgdown
 
 jobs:
   build:
+
+    strategy:
+      matrix:
+        version:
+          - 'release'
+          - 'devel'
+
+    name: pkgdown site build (${{ matrix.version }})
+
     runs-on: ubuntu-latest
     # Only restrict concurrency for non-PR jobs
     concurrency:
-      group: pkgdown-${{ github.event_name != 'pull_request' || github.run_id }}
+      group: pkgdown-${{ github.event_name != 'pull_request' || github.run_id }}-${{ matrix.version }}
     env:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
What is the difference between using a `GITHUB_PAT` vs a `GH_TOKEN` and why make this switch here?